api/v1/api.go

```diff
@@ -77,18 +81,160 @@ type postIsiExportResp struct {
 type IsiExport struct {
 	ID    int      `json:"ID"`
 	Paths []string `json:"paths"`
+	Clients []string `json:"clients"`
 }
 
 type getIsiExportsResp struct {
 	ExportList []*IsiExport `json:"exports"`
 }
 
+type isiThresholds struct {
```

Comment:
A better term than public/private that I found was exported/not-exported.  Returning a non-exported struct means that the receiver can access all the exported fields, but can't create their own variables of that type.  I think that is the desired behavior in this case.  The receiver can access all the data, but not create their own papi response struct.  This is probably why goxtremio used this approach.